### PR TITLE
Added missing NxCollapsibleItemsChild test - RSC-778

### PR DESCRIPTION
**Pull request by entroform**
https://issues.sonatype.org/browse/RSC-778

### CODE REVIEW

File: lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx [hunk 1]
```diff
@@ -26,6 +26,11 @@ describe('NxCollapsibleItems', function() {
     expect(NxCollapsibleItems).toBe(NxTreeView);
   });
 
+  it('has NxCollapsibleItemsChild that is aliased as NxTreeViewChild', function() {
+    expect(NxCollapsibleItemsChild).toBe(NxTreeViewChild);
```
Comment (rpokorny) [inline lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx:30]: `NxCollapsibleItemsChild` isn't publicly exported, so it doesn't need to be tested and shouldn't be the one mentioned in the test name

Comment (entroform) [reply lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx:30]: > `NxCollapsibleItemsChild` isn't publicly exported, so it doesn't need to be tested and shouldn't be the one mentioned in the test name

Oh it is publicly exported. We added the Child sub-component on the rename NxTreeView ticket and we still export the individual component so it's not a breaking change (I think).

Comment (rpokorny) [reply lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx:30]: > Oh it is publicly exported. We added the Child sub-component on the rename NxTreeView ticket and we still export the individual component so it's not a breaking change (I think).

`NxTreeViewChild` is still exported but `NxCollapsibleItemChild` isn't.  It is only externally available as `NxCollapsibleItems.Child`

File: lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx [hunk 2]
```diff
@@ -24,10 +24,6 @@ describe('NxCollapsibleItems', function() {
 
   it('is aliased as NxTreeView', function() {
     expect(NxCollapsibleItems).toBe(NxTreeView);
-  });
-
-  it('has NxCollapsibleItemsChild that is aliased as NxTreeViewChild', function() {
-    expect(NxCollapsibleItemsChild).toBe(NxTreeViewChild);
```
Comment (rpokorny) [inline lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx]: This isn't quite what I meant.  We want `expect(NxCollapsibleItems.Child).toBe(NxTreeViewChild);`

Comment (entroform) [reply lib/src/components/NxCollapsibleItems/__tests__/NxCollapsibleItems.test.tsx]: > This isn't quite what I meant. We want `expect(NxCollapsibleItems.Child).toBe(NxTreeViewChild);`

Thanks, updated it here: 0df577d035319b98ebd84048cd7bf2d151392346

**Review by rpokorny — APPROVED** [review]

**Pull request merged**